### PR TITLE
Clean up

### LINKED ISSUE #1: Feature: Iteration One - User Story
**Describe the User Story associated with this feature**
-User should be able to visit the webpage and see a randomized art piece.
-User should see three buttons home, favorites, and randomize

**Describe the acceptance criteria**
-User should successfully see all the expected elements on page.

**Any additional notes**

### LINKED ISSUE #2: Feature - Iteration Two - User Story
**Describe the User Story associated with this feature**
-User Should be able to click the randomize button to see a new a new piece displayed.
-User should either see error handling or skip to the next piece when no image url is provided for the art piece. 
**Describe the acceptance criteria**

**Any additional notes**

### LINKED ISSUE #3: Feature - Iteration Three - User Story
**Describe the User Story associated with this feature**
- User should be able to click the favorites button to add an art piece to their favorites section.
- User should be able to click the favorites tab to take them to a dashboard that displays all favorites. 
- User should be able to click the home button to return to the main dashboard. 
- 
**Describe the acceptance criteria**
-User should be able to see all successfully favorites art pieces. 

**Any additional notes**

### PR DESCRIPTION
**Pull request by nbalmoori**
# Description
This branch refactors the application's code to delete any unused files and to make sure code follow's style guidelines.

## Issues

Closes #1 
Closes #2 
Closes #3 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [ ] console.log()
- [ ] dev tools
- [x] functionality check through web browser
- [] tested by viewing the markdown file preview
- [x] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [ ] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

**Review by Jlavarine — APPROVED** [review]
Much more organized. Great job moving all the CSS to a styling folder.

File: README.md [hunk 1]
```diff
@@ -1,46 +1,38 @@
-# Getting Started with Create React App and Redux
+# Reaction Gallery
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+## Overview
```
Comment (whiteheadol) [inline README.md:3]: I can add to the readme!

File: README.md [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Goals:
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Technologies:
 
-### `npm test`
+- React
+- Redux / Redux Toolkit
```
Comment (whiteheadol) [inline README.md:22]: Heck yeah!

**Review by whiteheadol — APPROVED** [review]

**Review by DanielN88 — APPROVED** [review]
Everything looks good, the styling folder was a good addition. Ready for a merge.

**Pull request merged**